src/webview/leetCodeResultProvider.ts

```diff
@@ -34,19 +37,68 @@ class LeetCodeResultProvider implements Disposable {
         }
     }
 
-    private async provideHtmlContent(result: string): Promise<string> {
-        return `<!DOCTYPE html>
-            <html lang="en">
+    private parseResult(raw: string): IResult {
+        raw = raw.concat("  √ "); // Append a dummy sentinel to the end of raw string
```

Comment:
What will happen if we don't append `√`?

Reply:
The RegEx captures anything in between two `  √ `. If we don't append it, the last section (normally `stdout`) won't be captured.

Worth to note that, with this trick, messages like `[WARN] Failed to get memory percentile` will be captured as one part of `Your runtime beats 81.3 % of python submissions`, and will also be printed out even it does not starts with `  √ `.

Reply:
Hmm... still feeling the logic here is too tricky for maintaince. 

It's ok to merge for now.